daemon/logger/journald_semistruct/journald_semistruct.go

```diff
@@ -0,0 +1,140 @@
+// +build linux
+
+// Package journald provides the log driver for forwarding server logs
+// to endpoints that receive the systemd format.
+package journald_semistruct
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/Sirupsen/logrus"
+	cp "github.com/andyleap/parser"
+	sp "github.com/awakenetworks/semistruct-parser"
```

Comment:
be careful with these aliases, it can make the code harder to understand. semistruct isn't terrible to read either.